src/tv2-common/inewsConversion/converters/ParseCue.ts

```diff
@@ -722,18 +722,23 @@ function parseTargetEngine(
 	}
 
 	const graphicConfig = config.showStyle.GFXTemplates.find(
-		tmpl =>
-			tmpl.INewsCode.toUpperCase() === code?.toUpperCase() && tmpl.INewsName.toUpperCase() === iNewsName.toUpperCase()
+		template =>
+			template.INewsCode.toUpperCase() === code?.toUpperCase() &&
+			template.INewsName.toUpperCase() === iNewsName.toUpperCase()
 	)
 
+	const graphicDesignConfig = code
+		? config.showStyle.GfxDesignTemplates.find(template => template.INewsName.toUpperCase() === code?.toUpperCase())
```

Comment:
If this were to work, shouldn't we compare `INewsName`s?
```suggestion
		? config.showStyle.GfxDesignTemplates.find(template => template.INewsName.toUpperCase() === INewsName?.toUpperCase())
```

Reply:
I'll take a look at it, right now it just follows the structure used for the graphicConfig above it.

Reply:
I'm sorry i didn't understand the mistake until i could actually look at it today, i fixed it in the newest commit [3b084c1](https://github.com/tv2/sofie-blueprints-inews/pull/153/commits/3b084c140ddef3c5ba4bc5c2c862edd1ebab1415).